pep-0692.rst

```diff
@@ -424,16 +393,16 @@ hinting ``**kwargs`` using a ``TypedDict`` as described in this PEP will not
 require refactoring and function body and function invocations could be
 appropriately type checked.
 
-Another useful pattern that justifies using and typing ``**kwargs`` as proposed
-is when the function's API should allow for optional keyword arguments that
-don't have default values.
-
 Moreover, using ``**kwargs`` with more precise type hinting would allow for
 less code and less copy pasting in cases when a module contains a bunch of
 functions that expect similar keyword arguments. Using ``**kwargs`` with a
 ``TypedDict`` as propsed in this PEP is much more concise than explicitly
 repeating the keyword arguments every time. 
 
+Another useful pattern that justifies using and typing ``**kwargs`` as proposed
+is when the function's API should allow for optional keyword arguments that
+don't have default values.
+
```

Comment:
I know you just moved this paragraph -- but it's also added to the Motivation section. Why the duplication?

Reply:
That's a good point, I reworked the "Intended Usage" section so that the examples detailed in the "Motivation" section are not repeated - there is only a short summary of those to make the transition to "better tools for the job" paragraph smoother.